Code/GraphMol/DetermineBonds/catch_tests.cpp

```diff
@@ -0,0 +1,124 @@
+#include "RDGeneral/test.h"
+#include "catch.hpp"
+#include <RDGeneral/Invariant.h>
+#include <GraphMol/RDKitBase.h>
+#include <GraphMol/FileParsers/FileParsers.h>
+#include <GraphMol/FileParsers/MolSupplier.h>
+#include <GraphMol/DetermineBonds/DetermineBonds.h>
+#include <GraphMol/SmilesParse/SmilesParse.h>
+#include <GraphMol/AddHs.cpp>
+#include <iostream>
+#include <fstream>
+
+using namespace RDKit;
+
+TEST_CASE("Determine Connectivity") {
+    SECTION("Van der Waals") {
+        unsigned int numTests = 39;
+        for (unsigned int i = 0; i < numTests; i++) {
+            std::string rdbase = getenv("RDBASE");
+            std::string fName = rdbase +
+            "/Code/GraphMol/DetermineBonds/test_data/"
+            + "test" + std::to_string(i) + ".xyz";
+            std::unique_ptr<RWMol> mol(XYZFileToMol(fName));
+            REQUIRE(mol);
+            std::string smiles = mol->getProp<std::string>("_FileComments");
+            std::unique_ptr<RWMol> orig(SmilesToMol(smiles));
+            REQUIRE(orig);
+
+            determineConnectivity(*mol, false, 0);
```

Comment:
Don't provide the charge argument here if it's not being used:
```suggestion
            determineConnectivity(*mol, false);
```